packages/integration-sdk-runtime/src/api/index.ts

```diff
@@ -61,6 +64,10 @@ export function createApiClient({
     opts.signAwsV4 = { credentials: defaultProvider({ maxRetries: 3 }) };
   }
 
+  if (lambda) {
+    opts.Lambda = lambda;
+  }
```

Comment:
Is the intention to have the managed runtime create and pass this in? I guess I expected the SDK to create the `Lambda` client

Reply:
That was the intention. I was hoping to keep most of the `aws-sdk` things out of the SDK, as for the open-source graph projects, they aren't really needed. But what's your thoughts on why you expected it here?

Reply:
Good point. Maybe we should actually just allow `alphaOptions` to be passed in to this function. Then we could remove `isLambdaUrl()` and all the stuff around `signAwsV4` from this project, which would be idea.

And I guess the `signAwsV4` stuff isn't even necessary anymore anyway

Reply:
This way you wouldn't even need to explicitly import `@aws-sdk/client-lambda`